modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -159,6 +163,16 @@ The API console request panel component shows a panel where users can test an AP
 * *Headline Font Family (Optional)*: Set the font family of the headline.
 * *Error Color (Optional)*: Set the color of error messages.
 
+== API Details Button
+
+The API details button enables users to access details about an API.
```

Comment:
Use the API details button to access details about an API. (Avoid passive voice.)

Reply:
The ACM user (reader of this doc) configures the button with the options described here and adds it to the ACM portal. Then the consumer developers using the portal can click the button to access details about an API. Clarified as "The API details button component provides portal users with details about an API."